content/includes/desktop-linux-launch.md

```diff
@@ -2,7 +2,7 @@ To start Docker Desktop for Linux:
 
 1. Open your **Applications** menu in Gnome/KDE Desktop and search for **Docker Desktop**.
 
-2. Select **Docker Desktop** to start Docker. <br> The Docker menu ({{< inline-image src="images/whale-x.svg" alt="whale menu" >}}) displays the Docker Subscription Service Agreement.
+2. Select **Docker Desktop** to start Docker. <br> The Docker Subscription Service Agreement displays
```

Comment:
Seems this got cut off

<img width="750" alt="image" src="https://github.com/user-attachments/assets/82657523-9bce-465d-8998-ea1dec067f38">

Reply:
OK, then it was just missing a period at the end of it. Also we don't need the `<br>` here

```suggestion
2. Select **Docker Desktop** to start Docker.

   The Docker Subscription Service Agreement displays.
```